File: snippet/examples/kotlin/har.kt
val client = OkHttpClient()

val mediaType = MediaType.parse("application/json")
val body = RequestBody.create(mediaType, "{\"x\":1, \"y\":2, \"hex\":\"#102932\"}")
val request = Request.Builder()
  .url("http://localhost:8081/api/pixels")
  .post(body)
  .addHeader("Content-Type", "application/json")
  .addHeader("User-Agent", "curl/7.68.0")
  .addHeader("Accept", "*/*")
  .build()

val response = client.newCall(request).execute()

File: snippet/examples/csharp/har.cs
var client = new RestClient("https://place.willx86.com/api/pixels");
var request = new RestRequest(Method.POST);
request.AddHeader("Content-Type", "application/json");
request.AddHeader("User-Agent", "curl/7.68.0");
request.AddHeader("Accept", "*/*");
request.AddParameter("application/json", "{\"x\":1, \"y\":2, \"hex\":\"#102932\"}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
